File: Calculators/Science/Physcial_Science/OHM_Calculator.py
# Instructions:
# Just run the file and answer the questions!
# --------------------------------------------------------------------

while True:
    exchange = input("What are you trying to find (Voltage, Current(Ampage), or Resistance)\n")
    if "vol" in exchange.lower():
        print("")
        amps = input("What is the amount of amps?\n")
        print("")
        resistance = input("What is the amount of resistance?\n")
        print("")
        amps_int = float(amps)
        resistance_int = float(resistance)
        voltage = amps_int * resistance_int
        print("The voltage is:",voltage,"volts\n")
        continue_question = input("Would you like to do another calculation?\n")
        if "y" in continue_question.lower():
            for i in range(20):
                print("\n")
            continue
        else:
            break
    elif "cur" in exchange.lower():
        print("")
        volts = input("What is the amount of volts?\n")
        print("")
        resistance = input("What is the amount of resistance?\n")
        print("")
        volts_int = float(volts)
        resistance_int = float(resistance)
        current = volts_int / resistance_int
        print("The current(Ampage) is:",str(current)+"amps\n")
        continue_question = input("Would you like to do another calculation?\n")
        if "y" in continue_question.lower():
            for i in range(20):
                print("\n")
            continue
        else:
            break
    elif "res" in exchange.lower():
        print("")
        volts = input("What is the amount of volts?\n")
        print("")
        amps = input("What is the amount of amps?\n")
        print("")
        volts_int = float(volts)
        amps_int = float(amps)
        resistance = volts_int / amps_int
        resistance_str = str(resistance)
        print("The resistance is:",resistance_str,"(Ohms)")
        continue_question = input("Would you like to do another calculation?\n")
        if "y" in continue_question.lower():
            for i in range(20):
                print("\n")
            continue
        else:
            break
    else:
        print("Please put in a valid input!")
        continue

File: Calculators/Science/Physcial_Science/OHM_Calculator.java
// Instructions:
// Just run the file and answer the questions!
//--------------------------------------------------------------------
package Calculators.Science.Physcial_Science;

import java.util.Scanner;

public class OHM_Calculator {
	
	private static Scanner sc;
    private static boolean isContinuing;
    
    private static double ampage;
    private static double voltage;
    private static double resistance;

	public static void main( String[] args )
    {
        sc = new Scanner( System.in );
        isContinuing = true;

        ampage = 0;
        voltage = 0;
        resistance = 0;
        
        while ( isContinuing )
        {
            System.out.println(
                "What are you trying to find: Voltage, Current, or Resistance?" );
            String input = sc.next();

            switch ( input )
            {
            case "Voltage":
            	System.out.println("What is the ampage?");
            	ampage =  sc.nextDouble();
            	System.out.println("What is the resistance?");
            	resistance = sc.nextDouble();
            	
            	voltage = ampage * resistance;
            	System.out.println("The voltage is: " + voltage + " volts.");

            	break;

            case "Current":
            	System.out.println("What is the voltage?");
            	voltage =  sc.nextDouble();
            	System.out.println("What is the resistance?");
            	resistance = sc.nextDouble();
            	
            	ampage = voltage / resistance;
            	System.out.println("The current is: " + voltage + " amps.");
            	
            	break;

            case "Resistance":
            	System.out.println("What is the ampage?");
            	ampage =  sc.nextDouble();
            	System.out.println("What is the voltage?");
            	voltage = sc.nextDouble();
            	
            	resistance = voltage / ampage;
            	System.out.println("The resistance is: " + resistance + ".");
            	
            	break;

            default:
                System.out.println( "Invalid input." );
                
                break;
            }
            
            System.out.println( "Would you like to continue? Y/N" );
            String userInputToContinue = sc.next();
            
            if( userInputToContinue == "Y")
            {
                isContinuing = true;
            }
            else
            {
            	if( !(userInputToContinue == "N") )
            	{
                    System.out.println(
                        "Invalid input, this program will now be terminated." );
            	}
            	
            	isContinuing = false;
            }
        }
    }
}